cypress/e2e/tests/pages/users-and-auth/users.spec.ts

```diff
@@ -66,7 +66,7 @@ describe('Users', { tags: '@adminUser' }, () => {
 
     // usersPo.goTo();
     usersPo.waitForPage();
-    usersPo.list().elementWithName(userBaseUsername).should('be.visible');
+    usersPo.list().elementWithName(userBaseUsername).should('exist');
```

Comment:
I encountered issues with tests failing because elements being rendered outside of the viewport

```Timed out retrying after 60000ms: expected '<tr.main-row>' to be 'visible'

This element <tr.main-row> is not visible because its content is being clipped by one of its parent elements, which has a CSS property of overflow: hidden, scroll or auto

```

Using `exist` fixes the problem.